src/js/tooltip.js

```diff
@@ -455,6 +461,23 @@ class Tooltip {
 		this.tooltipRect = rect;
 	}
 
+	_getTooltipPosition() {
+		const { position, positionS, positionM, positionL, positionXl } = this.opts;
+		const currentBreakpoint = Tooltip._getCurrentLayout();
+		switch (currentBreakpoint) {
+			case 'S':
+				return positionS || position;
+			case 'M':
+				return positionM || positionS || position;
+			case 'L':
+				return positionL || positionM || positionS || position;
+			case 'XL':
```

Comment:
@notlee I noticed that o-grid returns the XL code as uppercase, but the options parser for o-tooltip returns Xl, the mismatch could be confusing. Outside the scope of this PR tho..